code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2011,8 +2011,8 @@
 	taste_description = "plastic"
 
 /datum/reagent/glitter
-	name = "Generic Glitter"
-	description = "If you can see this description, contact a coder."
+	name = "light pink glitter"
+	description = "light pink sparkles that get everywhere."
 	color = "#FFFFFF" //pure white
```

Comment:
the comment confuses me. I think it should say the base color of the decal is light pink

Reply:
It confused me too, its a holdover from before I started working on it. I think its applying a white filter onto a sprite that was darker pink/purple to end up with a light pink sprite. I will tinker with it this evening to figure out what is going on.

Reply:
After further tests, it seems that this line doesn't affect the colour but removing it stops a number of recipes from working so I am changing the comment and leaving it alone.